django/tasks/register.py

```diff
@@ -0,0 +1,28 @@
+class AlreadyRegistered(Exception):
+    pass
+
+
+class TaskDoesNotExist(Exception):
+    pass
+
+
+class TaskRegistry(object):
+    def __init__(self):
+        self._registry = {}
+
+    def register(self, task):
+        if task.name in self._registry:
```

Comment:
Does that mean the task name must be unique across queues? If so, I don't like that.